website/docs/d/satellite_cluster_worker_pool_zone_attachment.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+subcategory: "Satellite"
+layout: "ibm"
+page_title: "IBM : satellite_cluster_worker_pool_zone_attachment"
+description: |-
+  Get information about IBM Cloud satellite cluster worker pool zone attachment.
+---
+
+# ibm\_satellite_cluster_worker_pool_zone_attachment
```

Comment:
change to ibm_satellite_cluster_worker_pool_zone_attachment

Reply:
updated